src/main/java/model/dataaccess/BrandDAO.java

```diff
@@ -1,9 +1,123 @@
 package model.dataaccess;
 
+import model.entities.Brand;
+import org.hibernate.HibernateException;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+import org.hibernate.cfg.Configuration;
 
+import java.util.ArrayList;
+import java.util.List;
+import static main.MainClass.factory;
 
 
 public class BrandDAO implements IBrandDAO{
 
 
-}
+
+    public BrandDAO(){}
+
+    public static void getConfig(){
+        try{
+            factory = new Configuration().
+                    configure().
+                    addPackage("model.entities"). //add package if used.
+                    addAnnotatedClass(Brand.class).
+                    buildSessionFactory();
+        }catch (Throwable ex) {
+            System.err.println("Failed to create sessionFactory object." + ex);
+            throw new ExceptionInInitializerError(ex);
+        }
+    }
+
+    public ArrayList<Brand> getAll() {

```

Comment:
getAll poate returna direct List in loc de ArrayList si atunci nu mai ai nevoie de cast la return